mapit/views/areas.py

```diff
@@ -394,10 +394,13 @@ def areas_by_point(request, srid, x, y, bb=False, format=''):
 
     q = query_args(request, format)
 
+    max_within = getattr(settings, 'MAPIT_WITHIN_MAXIMUM', 0)
     try:
         within = float(request.GET.get('within', 0))
+        if within < 0 or within > max_within:
```

Comment:
This will unfortunately break the  feature entirely if `within` has a value > 0 and `MAPIT_WITHIN_MAXIMUM` is undefined (because `max_within` defaults to zero).

Reply:
Yes, I don't think it should work unless you specify a maximum (for the reason you stated, that it would be possible to overload by requesting all areas with a huge distance). So it's a way of switching the feature off too.